CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to xeroclient
+
+Thank you for your interest in xeroclient! Only with all of our help can we improve and move open source software forward.
+
+## Goal
+
+The goal of xeroclient (the software project) is to provide a clean, framework-agnostic PHP library to interface with the [Xero API](https://developer.xero.com/).
+
+## Code of Conduct
+
+* We shall be respectful when we post in the issue queue, pull request queue, in our code, in commit messages. Communication that is disrespectful on the basis of race, gender, sexual orientation, religion, ethnicity, language, age, physical capabilities, technical skill, or contribution experience are not welcome on this project.
+* We shall welcome contribution and recognize that the most valuable contributions come from those of us from different backgrounds and experiences. Note that accounting is a global industry with different complexities depending on the locality in which the software is used.
+* We shall try to reach a consensus on decisions in the issue queue and recognize that some times a decision cannot be unanimously reached. The project maintainer will make a decision based on the current state of an issue or pull request.
+* We shall give attribution to code and non-code contributions.
+* We shall respect those who fork the repository and adapt xeroclient to their own needs.
+
+## Accountability
+
+* "mradcliffe" will make decisions regarding code and software architecture underneath the `mradcliffe/xeroclient`. A fork maintainer is responsible for their actions underneath their own remote repository.
+* "mradcliffe" will close and delete any comment that does not adhere to the Code of Conduct. Code and commit messages that do not adhere to the Code of Conduct cannot be modified, but should be addressed publically.
+* Behavior that is unacceptable by "mradcliffe" can and should be reported to the Drupal Community Working Group even though this project is not related to Drupal. This is the closest governing body to address accountability for "mradcliffe" as they have leadership roles in the Drupal community.
+
+## Getting Started
+
+See the [README](./README.md) for usage and installation instructions.
+
+### Reporting an issue
+
+Reporting an issue is one of the most important means of contribution. Creating an issue is great!
+
+1. Search the issue queue for one or more key words that may be related to a bug, task, or feature that you are interested in.
+2. Create the issue using the [issue template](./ISSUE_TEMPLATE.md).
```

Comment:
Testing this led me to a message that I was using the "old way" and linked to [About issue and pull request templates](https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates).